cou/steps/plan.py

```diff
@@ -160,6 +161,23 @@ async def post_upgrade_sanity_checks(analysis_result: Analysis) -> None:
     except ApplicationError:
         messages.append("Detected ceph 'noout' set, please unset noout for ceph cluster.")
 
+    for app in ceph.get_mon_apps(analysis_result.apps_control_plane):
```

Comment:
(non-blocking) ideally, the same `verify_cloud` pattern can be used for this function, but that's more like a small refactoring:

```python
async def post_upgrade_sanity_checks():
    _post_verify_a()
    _post_verify_b()
```

Reply:
Agreed, let's leave such refactoring to a future PR